src/bootstrap/config.rs

```diff
@@ -2089,6 +2098,10 @@ impl Config {
     }
 }
 
+fn is_llvm_config_identical(from_ci: &LLVMConfig, current: &LLVMConfig) -> bool {
+    from_ci == current
+}
```

Comment:
looks like this used to be a larger function before you replaced it with derive(Eq)? i think you can get rid of it now and just use `from_ci == current` inline.

Reply:
It wasn't, I just wanted to make this super explicit. But `==` is fine too :)